accounts/ethkey/src/brain.rs

```diff
@@ -18,7 +18,7 @@ use keccak::Keccak256;
 use super::{KeyPair, Generator, Secret};
 use parity_wordlist;
 
-/// Simple brainwallet.
+/// Simple brainwallet.s
```

Comment:
Huch? :D

Reply:
xD
that was for testing if pipeline will run with Rust files changed